test/index.test.ts

```diff
@@ -29,7 +29,7 @@ describe("Sign transaction function", () => {
 
 describe("Recover the address function of a transaction", () => {
   const lsp6Signer = new LSP6Signer();
-  const signingAddress = "0x2b389f8EB52D16A105e02165a2AC1450461A237b";
+  const signingAddress = "0x2b389f8EB52D16A105e02165a2AC1450461A237b"; //private key
```

Comment:
Is this a signing key or an address? We can remove the comment and change the name of the variable so it is accurate